### DISCUSSION TITLE
online leaderboard

### DISCUSSION THREAD

**Discussion by whyvn** [agateau/pixelwheels#229] [Ideas]
thought it would be cool if you could submit scores to an online leader board. you could host on GitHub pages if you wanna keep things connected with this repo.

**Comment by agateau** [comment]
Hi, this is something I would like to add in the future, but I don't see any way to prevent cheating, given that the code is opensource :/

**Comment by whyvn** [comment]
Yeah, i was thinking about that and earlier and i think that the only way it will be moderated is by a human. We could use an external leader board such as [speedrun.com](https://speedrun.com/) or use the same approach to a self hosted ones (save replay files of someones personal bests so if they submit a human mod will be able to verify it)

**Reply by whyvn** [reply]
its not that important though.
